README.md

```diff
@@ -90,6 +96,43 @@ $ tail -f /var/log/td-agent/td-agent.log
   * to use short hostname, set `hostname -s` for this option on linux/mac.
   * [default] `hostname`
 
+### About Windows
+
+Default `command` in Windows below is complicated, but you can fix `keys` without fixing `command`, unless you specify a key other than the default.
+
+`````powershell
+powershell -command "Get-Process -IncludeUserName
+ | ?{$_.StartTime -ne $NULL -and $_.CPU -ne $NULL}
+ | Select-Object -Property StartTime,UserName,SessionId,Id,CPU,WorkingSet,VirtualMemorySize,HandleCount,ProcessName
+ | %{
+  $_.StartTime = $_.StartTime.ToString(
+  'ddd MMM dd HH:mm:ss yyyy',
```

Comment:
This particular format seems non-standard. It will produce
something like this:

```
Mon May 17 00:51:23 2021
```

For comparison, on Unix `date` produces (on C locale):

```bash
$ date
Mon 17 May 2021 10:21:37 AM JST
```

Or with powershell:

```powershell
PS> get-date
Monday, May 17, 2021 1:22:16 AM
```

Reply:
Thank you for the advice.

I use `ddd MMM dd HH:mm:ss yyyy` as the format to match the regular expression below.

https://github.com/daipom/fluent-plugin-watch-process/blob/621b59de79e7dbb706812804daa1f84933d4c5d5/lib/fluent/plugin/in_watch_process.rb#L130-L131

I have confirmed the format of `ps`  on mac and Redhat in the locale en-US.
It is something like `Mon May 17 00:27:59 2021`.

In `date` command in en-US locale, it is somthing like `Mon May 17 15:07:39 JST 2021`.

I don't know the details, but this format may actually be  common for the en-US locale.

Reply:
Since Windows watcher use csv format for parsing,  it does not use regular expressions.
So  as long as `Time.parse` works well, we can fix this format in Windows logic.

Reply:
I have not fixed the input format yet,
but I have fixed the output format to be the same as Linux and mac in https://github.com/daipom/fluent-plugin-watch-process/pull/2/commits/25919cd08fef97f0b9617a7976a17e5af6876856.

Reply:
I think we should not fix this input format `ddd MMM dd HH:mm:ss yyyy`
since this is the same format as UNIX like in the locale `en-US`.